Spanner/src/Backup.php

```diff
@@ -186,6 +186,44 @@ public function create($database, \DateTimeInterface $expireTime, array $options
         return $this->resumeOperation($operation['name'], $operation);
     }
 
+    /**
+     * Create a copy of an existing backup in Cloud Spanner.
+     *
+     * Example:
+     * ```
+     * $spanner = new SpannerClient();
+     * $instance = $spanner->instance('destination-instance-id');
+     * $destBackup = $instance->backup('new-backup-id');
+     * $operation = $backup->createCopy($destBackup, new \DateTime('+7 hours'));
+     * ```
+     *
+     * @param string $newBackup The backup object that needs to be created as a copy.
```

Comment:
The docblock here is wrong

```suggestion
     * @param Backup $newBackup The backup object that needs to be created as a copy.
```